test/units/modules/network/mlnxos/test_mlnxos_linkagg.py

```diff
@@ -0,0 +1,128 @@
+#
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.compat.tests.mock import patch
+from ansible.modules.network.mlnxos import mlnxos_linkagg
+from units.modules.utils import set_module_args
+from .mlnxos_module import TestMlnxosModule, load_fixture
+
+
+class TestMlnxosLinkaggModule(TestMlnxosModule):
+
+    module = mlnxos_linkagg
+
+    def setUp(self):
+        super(TestMlnxosLinkaggModule, self).setUp()
+        self.mock_get_config = patch.object(
+            mlnxos_linkagg.MlnxosLinkAggModule,
+            "_get_port_channels")
+        self.get_config = self.mock_get_config.start()
+
+        self.mock_load_config = patch(
+            'ansible.module_utils.network.mlnxos.mlnxos.load_config')
+        self.load_config = self.mock_load_config.start()
+
+    def tearDown(self):
+        super(TestMlnxosLinkaggModule, self).tearDown()
+        self.mock_get_config.stop()
+        self.mock_load_config.stop()
+
+    def _execute_module(self, failed=False, changed=False, commands=None, sort=True):
```

Comment:
Wondering why this method is defined in each unit test file, because this is already defined in mlnxos_module.py

Reply:
Thanks for the notice trishnaguha:
That was a leftover from the evolution of the test, first stage was to override the loadfixtures.
Then I decided to call loadfixtures directly, and I forgot to remove this function.
will remove and re-push again